packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -202,6 +203,9 @@ class ProductFullDetail extends Component {
                     </h2>
                     <strong>{product.sku}</strong>
                 </section>
+                <section className={classes.stickyFooter}>
+                    <ProductStickyFooter addToCartHandler={this.addToCart} />
```

Comment:
Another option is `onAddToCart`, which more closely conforms with component event handlers like `onClick`, `onChange`, etc.